akka-typed/src/main/scala/akka/typed/ScalaProcess.scala

```diff
@@ -0,0 +1,538 @@
+/**
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com/>
+ */
+package akka.typed
+
+import scala.concurrent.duration._
+import akka.{ actor ⇒ a }
+
+object ScalaProcess {
+  import ScalaProcess._
+
+  class RetriesExceeded extends RuntimeException
+
+  import language.implicitConversions
+  /**
+   * This implicit expresses that operations that do not use their input channel can be used in any context.
+   */
+  private implicit def nothingIsSomething[T, U](op: Operation[Nothing, T]): Operation[U, T] = op.asInstanceOf[Operation[U, T]]
+
+  /*
+   * Convenient access to the core operations, will be even nicer with Dotty implicit function values
+   */
+  sealed trait OpDSL extends Any {
```

Comment:
Nope, to make it absolutely impossible to call any methods on it—because the only value that will ever exist is `null`.